package/test/__snapshots__/index.test.ts.snap

```diff
@@ -1,26 +1,75 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`requestHandler should handle highlighting 1`] = `
-[
-  {
-    "_highlightResult": {
-      "description": {
-        "fullyHighlighted": false,
-        "matchLevel": "full",
-        "matchedWords": [],
-        "value": "__ais-highlight__Football__/ais-highlight__ __ais-highlight__awesome__/ais-highlight__ Is Good For Training And Recreational Purposes",
```

Comment:
Why has this result changed? The `football` query is still in the test.

Reply:
Similar to some of the other keyword changes.. 
> Adding to createRandomProduct changed the database result. Shouldn't happen again.

Reply:
That only explains the *addition* (Carbonite gloves) to the snapshot. It doesn't explain the *deletion* (football) in it. I still don't understand why this test searches for "football" but doesn't expect "football" in the result?

Also, I find it naive to think that this won't happen again. Adding more features tomorrow will likely require more fiddling with `faker` to make those features testable.

Reply:
The reason the football query was dropped is because I wanted to test a query that highligted words in both the name and description. And with the updated faker seed, football wasn't meeting that requirement anymore.

With the revised code in `createRandomProduct` additional properties can be generated as long as they are added to the end of the function. These won't have the knock on effect of offsetting the seed. I've added a comment. 

I can't really think of a better way to have reproducable and semi-realistic test data. I'm open to suggestions.

Reply:
> The reason the football query was dropped

But the football query wasn't dropped, that's my whole point.  It's still right there in `index.test.ts`: https://github.com/data-envoy/postgres-searchbox/blob/5065a415993f3cb904c02c3b4ced9d01e9ce505b/package/test/index.test.ts#L309

> With the revised code in `createRandomProduct` additional properties can be generated as long as they are added to the end of the function. These won't have the knock on effect of offsetting the seed. I've added a comment.

Great.
 
> I can't really think of a better way to have reproducable and semi-realistic test data. I'm open to suggestions.

This is more than good enough for now.  One day I'm envisioning explicitly populating the data (we can run `faker` once and record the results), so we can see exactly what's in the corpus and how each query matches it.

Reply:
Ah I think I know what happened. For highlighting I changed the query to 'bespoke ball' to get highlighting in both name and desc.

I left it as football in the `should handle returning columns` test, because that test the search query is only testing that the right columns are being returned.

I appreciate an explination in the description would have been more helpful, I'll keep that in mind for future work!

> One day I'm envisioning explicitly populating the data (we can run faker once and record the results)

That sounds like a good plan :D

Reply:
OK.